frontend/src/pages/EditRules/EditRules.styles.scss

```diff
@@ -11,20 +11,19 @@
 .edit-rules-card {
 	width: 20rem;
 	padding: 1rem;
-}
 
-.content {
-	font-style: normal;
-	font-weight: 300;
-	font-size: 18px;
-	line-height: 20px;
-	display: flex;
-	align-items: center;
-	justify-content: center;
-	text-align: center;
-	margin: 0;
+	.content {
```

Comment:
@vikrantgupta25, this won’t completely fix the issue. The `.content ` styles from other places will still apply, and using nesting will only increase the specificity, overriding it. What we need to do is rename `.content` to something else, like `edit-rules-card__content` using the BEM naming convention. This will resolve the scoping issue since the new name is unique, whereas `.content` is widely used elsewhere in our code and in other libraries as well.

Reply:
We follow a principle of scoping styles to the parent container which increases the specificity of our styles. 
`.content` will be applied to any other un-intended area if and only if that follows a similar hierarchy. 

> .content is widely used elsewhere in our code and in other libraries as well.

As long as it is scoped properly to the parent container class name we won't face any issues. Adding global styles always causes issues so we should avoid doing that, which is being fixed in this PR. I haven't renamed the styles as didn't want the blast radius of this PR to be around alerts module as well. @ahmadshaheer  has been made aware of this and if needed we will fix that separately.